tests/Bridge/Doctrine/Orm/Extension/FilterEagerLoadingExtensionTest.php

```diff
@@ -270,6 +347,6 @@ public function testCompositeIdentifiers()
 
     private function toDQLString(string $dql): string
     {
-        return preg_replace('/\\r\\n|\\n/', '', str_replace('  ', '', $dql));
+        return preg_replace(['/\s+/', '/\(\s/', '/\s\)/'], [' ', '(', ')'], $dql);
```

Comment:
Actually this is unsafe... It might change whitespace within a SQL literal (string). Lol...